lib/canary/plugs.ex

```diff
@@ -23,8 +23,8 @@ defmodule Canary.Plugs do
   @doc """
   Load the given resource.
 
-  Load the resource with id given by `conn.params["id"]` and ecto model given by
-  opts[:model] into `conn.assigns.resource_name`.
+  Load the resource with id given by `conn.params[opt[:id]]` (`opts[:id]` defaults to "id") ecto model given
```

Comment:
Just a spelling mistake: should be opts[:id] rather than opt[:id].

 I also think it is clearer to say "given by `conn.params["id"]` (or `conn.params[opst[:id]]` if opts[:id] is specified)"